tracer/test/snapshots/RemotingTests.http.SchemaV0.verified.txt

```diff
@@ -0,0 +1,169 @@
+[
```

Comment:
Just a reminder for others, you can "visualise" the snapshots using https://andrewlock.github.io/DatadogSpanSnapshotViewer/ and entering the github url of a snapshot, e.g. https://github.com/DataDog/dd-trace-dotnet/blob/061aba64436e4dd2690d8d5ca3589031cac0673b/tracer/test/snapshots/RemotingTests.http.SchemaV0.verified.txt